### PR TITLE
improve formatting of Readme

### PR DESCRIPTION
**Pull request by IzzySoft**
resizing display of badges and screenshots, some slight formatting adjustments

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+<div align="center" markdown="1">The native, Android [Food Scanner App](https://play.google.com/store/apps/details?id=com.lifyzer) 🥝*
 
-![Healthy Food, Healthy Tomato](extras/assets/broccoli-healthy-food.svg)
+![Healthy Food, Healthy Tomato](extras/assets/broccoli-healthy-food.svg)</div>
 
 
 ## About The App 🤔
 
-* Lifyzer App uses OCR and barcode scanner technologies.
+<div align="center" markdown="1">Lifyzer App uses OCR and barcode scanner technologies.
```
Comment (pH-7) [inline README.md:10]: Oh... didn't know the `markdown` GitHub's attribute. But not sure it actually does something...

**Review by pH-7 — APPROVED** [review]
Looks good to me. Thanks for this! I really appreciate your help 🤗

File: README.md (hunk 1 above)
Comment (pH-7) [inline README.md:10]: We could use `style="align:center` which is the only HTML5's valid way

Comment (IzzySoft) [reply README.md:10]: according to the preview, it does :smile: Actually, I knew that from MarkdownExtra, and was surprised GH-flavored uses it as well.

Comment (IzzySoft) [reply README.md:10]: Feel free to build upon. I didn't want to introduce too much non-Markdown, so I used the old-fashioned approach (OK, call it "quick-and-dirty" :speak_no_evil:)

Comment (pH-7) [reply README.md:10]: No worries :)

**Pull request merged**